Dockerfile

```diff
@@ -1,11 +1,15 @@
 FROM golang:alpine as unwindtest
 
 ENV GO111MODULE=on \
-    CGO_ENABLED=0
+    CGO_ENABLED=0 \
+    GOFLAGS=-mod=mod
 
 COPY . /app/
 WORKDIR /app/
 
 RUN ls -al
 
+EXPOSE 8010
+RUN go install -v github.com/weihesdlegend/Vacation-planner
 RUN go build -v .
```

Comment:
Can we try to remove build if install already covers it?

Reply:
yeah sure.. I will try that and check